packages/klevu-ui/src/components/klevu-list/klevu-list.tsx

```diff
@@ -40,41 +40,14 @@ export class KlevuList {
 
   @Prop() noXPadding = false
 
-  @State() secondarySlotChildCount = 0
-
-  componentDidLoad() {
-    this.#checkSecondarySlotChildCount()
-  }
-
-  #checkSecondarySlotChildCount() {
-    const slot = this.el.shadowRoot?.querySelector("slot[name='secondary']") as HTMLSlotElement | null
-    this.secondarySlotChildCount = slot?.assignedElements().length ?? 0
-  }
-
-  @Listen("slotchange")
-  handleSlotChange() {
-    this.#checkSecondarySlotChildCount()
-  }
-
   #getContent() {
     return (
       <Fragment>
         {this.icon && <klevu-icon part="list-icon" class="icon" name={this.icon} />}
         {this.image && <img part="list-image" class="image" src={this.image} />}
         <div class="text" part="list-content">
-          <klevu-typography variant="body-s">
-            <slot name="primary"></slot>
-          </klevu-typography>
-          {this.secondarySlotChildCount > 0 && (
-            <klevu-typography
-              variant="body-s"
-              style={{
-                "--klevu-typography-color": "var(--klevu-color-neutral-7)",
-              }}
-            >
-              <slot name="secondary"></slot>
-            </klevu-typography>
-          )}
+          <slot name="primary"></slot>
```

Comment:
We lose the styling of content when dropping `klevu-typography`. But we could make this better. 

Lets make separate props with text so in default case there could be typography used that is fully replaced.

```suggestion
          <slot name="primary">
            <klevu-typography variant="body-s">{this.primaryText}</klevu-typography>
          </slot>
          <slot name="secondary">
            {this.secondaryText ? <klevu-typography variant="body-s" style="...">{this.secondaryText}</klevu-typography>
          </slot>
```

With this solution we get best of both worlds.

Reply:
since its a slot, the customer can paste their own styled text in it, dont you think its better to go with the bare approach?

Reply:
But then they have to know exactly what to put in the list in order to make it look good.

Reply:
Correct. I feel that is what is expected when we add content to a slot.
I have concerns with creating props just for using the styling, both slots and the props become duplicate.

As an alternate, can we add css to the klevu-list.css for slot. These will be overridden with whatever is passed into the slot.
`slot[name="primary"] {
  font-size: var(--klevu-size-m);
}
slot[name="secondary"] {
  font-size: var(--klevu-size-s);
}`

Reply:
Added the props, you can review now.